### PR TITLE
chore(sentry10): Move all users to Sentry 10

### PR DESCRIPTION
**Pull request by lynnagara**
This PR forces all users onto Sentry 10 and turns off support for the
deprecated single project based views. It will be merged and take effect
on May 31st.

It removes the "disable new visibility features" option for hosted
users.

From that point on, open source users who are not running the Snuba service
should use the `releases/9.1.x` branch until the next major version of Sentry
is released.

Needs to be deployed together with https://github.com/getsentry/getsentry/pull/2882

### CODE REVIEW

File: src/sentry/models/organization.py [hunk 1]
```diff
@@ -138,9 +138,6 @@ class Organization(Model):
             ), (
                 'require_2fa',
                 'Require and enforce two-factor authentication for all members.'
-            ), (
-                'disable_new_visibility_features',
-                'Temporarily opt out of new visibility features and ui',
```
Comment (markstory) [inline src/sentry/models/organization.py]: Should we bulk update the flags for organizations? Next time when we add a flag some users will have state in that bit.

File: src/sentry/static/sentry/app/data/forms/organizationGeneralSettings.jsx [hunk 2]
```diff
@@ -38,12 +38,6 @@ const formGroups = [
         label: t('Early Adopter'),
         help: t("Opt-in to new features before they're released to the public"),
       },
-      {
```
Comment (eurica) [inline src/sentry/static/sentry/app/data/forms/organizationGeneralSettings.jsx]: 👍 for getting rid of settings cruft!

File: src/sentry/models/organization.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/models/organization.py]: @markstory Good call, I think we should tackle this in Sentry 10 cleanups. I added back these lines here so we don't forget about it later.

**Review by markstory — APPROVED** [review]
:clap:

**Pull request merged**